tests/scripts/must-gather.sh

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env bash
```

Comment:
Instead of maintaining a copy of this script here, let's download this script on the remote machine at runtime.

Reply:
added